api/server/call_logs.go

```diff
@@ -21,18 +28,22 @@ func (s *Server) handleCallLogGet(c *gin.Context) {
 		return
 	}
 
-	// TODO this API needs to change to text/plain / gzip anyway, punting
-	// optimization, but we can write this direct to the wire, too... seems like
-	// we should write some kind of writev json thing for go since we keep
-	// hitting this :(
-	var b bytes.Buffer
-	b.ReadFrom(logReader)
-
-	callObj := models.CallLog{
-		CallID:  callID,
-		AppName: appName,
-		Log:     b.String(),
+	mimeTypes, _ := c.Request.Header["Accept"]
+
+	for _, mimeType := range mimeTypes {
```

Comment:
this isn't the behavior we want. an example: 

`Accept: application/xml, text/plain` will return a json blob. the user clearly can handle `text/plain` but it's not explicit whether they can handle json, we should return `text/plain` since it was specified and json was not.

One suggestion might be switch on exact `Accept` headers in the loop, since we _do_ want to treat them as a prioritized list, and only default to json if we don't find any of the types we're expected. the HTML spec pretty clearly defines the behavior we should expose here: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.1 and we also need to handle `q` values (which won't be switched on properly in this code); for `q` values, I don't think we need to worry about quality reduction [for now/ever], just to parse them properly.